metadata/metadata_service/proxy/neo4j_proxy.py

```diff
@@ -170,7 +170,7 @@ def _exec_col_query(self, table_uri: str) -> Tuple:
             last_neo4j_record = tbl_col_neo4j_record
             col = Column(name=tbl_col_neo4j_record['col']['name'],
                          description=self._safe_get(tbl_col_neo4j_record, 'col_dscrpt', 'description'),
-                         col_type=tbl_col_neo4j_record['col']['col_type'],
+                         col_type=tbl_col_neo4j_record['col']['col_type'] or tbl_col_neo4j_record['col']['type'],
```

Comment:
I am not sure if that's long term fix. Given we have bump the major version for databuilder, we should update the doc to make it more explicit.

Reply:
What is the migration plan to migrate existing data to newer version? People are using older version of databuilder just because of this issue (this has been asked a number of times on Slack and users are eventually downgrading to older version). 

I think this is the efficient fix to support existing data and at the same time migrate to newer version.

Reply:
It doesn't solve the issue when we have breaking changes.

Reply:
We should advise the users to upgrade to a newer version. Or what is the downside of doing so?

Reply:
I think the comment is that there isn't a published migration guide to migrate the data from the older version into the newer format?

Reply:
We add sth similar before (https://github.com/amundsen-io/amundsen/blob/main/docs/faq.md#how-to-migrate-from-amundsen-1x---2x) for upgrade/migration notice.